### PR TITLE
updated build.gradle, dependencies, and other stuff

### PR DESCRIPTION
**Pull request by TacoTheDank**
read commit notes i guess

### CODE REVIEW

File: Clover/app/build.gradle [hunk 1]
```diff
@@ -13,13 +13,13 @@ def getCommitHash = { ->
 }
 
 android {
-    compileSdkVersion 27
+    compileSdkVersion 28
     // update the travis config when changing this
```
Comment (andyklimczak) [inline Clover/app/build.gradle]: you'll want to do this comment and update `.travis.yml`

File: Clover/app/build.gradle [hunk 2]
```diff
@@ -87,7 +87,7 @@ android {
         dev {
             dimension "default"
             applicationId "org.floens.chan.dev"
-            resValue "string", "app_name", "Clover dev"
+            resValue "string", "app_name", "Clover Dev"
```
Comment (Floens) [inline Clover/app/build.gradle]: The old value was good.

File: Clover/app/build.gradle [hunk 3]
```diff
@@ -124,7 +124,7 @@ android {
 }
 
 dependencies {
-    def supportVersion = '27.1.0'
+    def supportVersion = '28.0.0-alpha3'
```
Comment (Floens) [inline Clover/app/build.gradle]: Please do only use stable versions.

**Comment by TacoTheDank** [comment]
aight gimme a sec

**Comment by TacoTheDank** [comment]
28.0.0 doesnt have any stable libraries yet so i guess ill keep it at 27.1.1

**Comment by TacoTheDank** [comment]
a lot of the dependencies have been updated tho so yeah

**Comment by Floens** [comment]
Oh and we can't update the target api yet see #353

File: Clover/app/build.gradle [hunk 4]
```diff
@@ -15,11 +15,11 @@ def getCommitHash = { ->
 android {
     compileSdkVersion 27
     // update the travis config when changing this
-    buildToolsVersion '27.0.3'
+    buildToolsVersion '27.1.1'
 
     defaultConfig {
         minSdkVersion 15
-        targetSdkVersion 25
+        targetSdkVersion 27
```
Comment (Floens) [inline Clover/app/build.gradle]: revert

**Comment by Floens** [comment]
Did you even test this? The app doesn't compile at all.
The gradle version is wrong, the build tools don't exist, you changed the event bus to another one, etc.
Instead of fixing this PR I'll update the dependencies in a separate commit.

**Comment by Floens** [comment]
0a56b7c95e518f06955e2df1162d02ff71d49cd6

**Comment by TacoTheDank** [comment]
sorry, a few days ago i realized i mixed up the build-tools and the compat library version schemes but forgot to fix it on this repo

**Pull request closed without merging**